src/schema/convert-edtr-io-state.tsx

```diff
@@ -334,7 +277,9 @@ export function convert(
       })
       return result
     }
+
     // compat: extract math formulas
+    //TODO: can not reproduce, happens elsewhere now, right? can be deleted?
```

Comment:
Some background information here: https://github.com/serlo/frontend/issues/186 (it's about the first warning)